.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "KubeEdge Website DevContainer",
+  "build": {
+    "dockerfile": "../docker/image/Dockerfile"
+  },
+  "workspaceMount": "source=${localWorkspaceFolder},target=/tmp/doc_repository,type=bind",
```

Comment:
working space is bound to the repository, that means user just start `yarn start` with optional argument if they need to. this provides the out-of-the-box experience for documentation developers. (pretty much same thing goes to the docker users, but they need to make sure the docker run optional arguments though.)